topics/git/git-commands/index.md

```diff
@@ -29,39 +29,53 @@ to track. In each repository, you can commit files and folders and have git
 remember what changes you have made. You can also push your changes so that
 people you're working with can see them.
 
-We'll be dividing the set of things you might want to do into "collaborative"
-and "private" actions for the sake of organization. In reality, these lines are
-a bit blurred.
+## Getting Yourself a Git Repository
 
-## "Private" Actions
-
-### I would like to know what in the world is going on:
+### Creating a new, local repo:
 
 {% highlight bash %}
-$ git status
+$ mkdir <my-repo-name>
+$ cd <my-repo-name>
+$ git init
 
-AND
+OR
 
-$ git log --pretty=oneline --abbrev-commit --decorate --graph --all
-# or without as much information,
-$ git log
+git init <my-repo-name>
 {% endhighlight %}
 
-Get used to running these commands after you run any other Git command.
+If you're not starting a new project but are making changes to a project that
+already exists, it makes more sense to "clone" the repo - that means to download
+a copy of a git repository stored somewhere else.
 
-### Creating a new, local repo:
+### Cloning a shared, upstream repo
 
-{% highlight console %}
-$ mkdir <my-repo-name>
-$ cd <my-repo-name>
-$ git init
+{% highlight bash %}
+# If your upstream is for example, a folder on AFS
+$ git clone <path to bare> [optional directory to create]
 
-OR
+# If your upstream is online, like on GitHub
```

Comment:
hmm I'm not quite sure how to phrase this in a better way, but it sounds a bit awkward